conf/messages.en

```diff
@@ -297,8 +297,10 @@ supplementary.eori = EORI number
 supplementary.eori.info = Enter the EORI number or business details
 supplementary.eori.hint = The number starts with a country code, for example FR for France, and is then followed by up to 15 digits
 
+supplementary.eori.nongb.error=EORI number must start with GB followed by 12 to 15 digits
```

Comment:
Could you check with James Watson (content manager of the team) about these messages?
If you already have, then ignore me :stuck_out_tongue:

Reply:
Verified initially with Monica. Now with James. He is happy to show only 1 error message instead of 2. So for both GB or min/max value, this is what should be displayed..
EORI number must start with GB followed by 12 to 15 digits
Will fix the code accordingly.

Reply:
Fixed